src/Microsoft.Tye.Hosting/TyeHost.cs

```diff
@@ -194,6 +194,13 @@ private IHost BuildWebApplication(Application application, HostOptions options,
                                     });
                             });
                     services.AddSingleton(application);
+                    services.AddScoped(sp =>
+                    {
+                        var server = sp.GetRequiredService<AspNetCore.Hosting.Server.IServer>();
+                        var addressFeature = server.Features.Get<AspNetCore.Hosting.Server.Features.IServerAddressesFeature>();
+                        string baseAddress = addressFeature.Addresses.First();
+                        return new System.Net.Http.HttpClient { BaseAddress = new Uri(baseAddress) };
+                    });
```

Comment:
Feels slightly overkill to make requests to the API. We can just call the function, right?

Reply:
I think I'll need some help here... how do I access ProcessRunner from razor page?